src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -23,8 +23,26 @@ public Drivetrain() {
     backLeftMotor = new TalonFX(RobotMap.mapDriveTrain.BACK_LEFT_MOTOR);
   }
 
+  /**
+   * Sets the velocity of the drivetrain motors.
+   * 
+   * @param rightVelocity The velocity to set for the right side of the drivetrain.
+   * @param leftVelocity The velocity to set for the left side of the drivetrain.
+   * @param rotationSpeed The rotation speed to apply to the drivetrain.
+   */
+  public void setDrivetrainSpeed(double rightVelocity, double leftVelocity, double rotationSpeed) {
```

Comment:
Why not just forward velocity and rotate velocity?

Reply:
That makes sence